src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IArgument.cs

```diff
@@ -2712,6 +2703,67 @@ void M1()
             VerifyOperationTreeAndDiagnosticsForTest<ElementAccessExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
 
+        [Fact]
+        public void DirectlyBindArgument_Attribute()
+        {
+            string source = @"
+[assembly: /*<bind>*/System.CLSCompliant(isCompliant: true)/*</bind>*/]
```

Comment:
Is there a tracking issue for attributes?

Reply:
tracked by this - https://github.com/dotnet/roslyn/issues/18198